### PR TITLE
Fix/prevent webpack from mangling node globals

### PR DESCRIPTION
**Pull request by tejohnso**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -36,9 +36,11 @@ initialize()
...
   .then(downloadQueue.checkStaleFiles)
   .then(()=>{
-    log.all("started", {
-      version: config.getModuleVersion()
-    }, null, config.bqTableName);
+    const version = config.getModuleVersion();
+
+    setTimeout(()=>{
+      log.all("started", {version}, null, config.bqTableName);
+    }, config.initialLogDelay);
```
Comment (santiagonoguez) [inline src/index.js]: I assume this timeout is a quick workaround some unpleasant problem with the log global.

File: src/config/config.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ let moduleVersion = "";
 module.exports = {
   secondMillis: 1000,
   moduleName: MODULE_NAME,
+  initialLogDelay: 5000,
```
Comment (stulees) [inline src/config/config.js]: @tejohnso 5 secs seems a lot, can 2 secs suffice?

File: src/index.js (hunk 1 above)
Comment (tejohnso) [reply src/index.js]: LM and Logger timing

File: src/config/config.js (hunk 2 above)
Comment (tejohnso) [reply src/config/config.js]: What's the concern? I don't see why it matters.

Comment (stulees) [reply src/config/config.js]: @tejohnso No major concern, just thinking in regards to order of logs in BQ as other logs may be sent within the 5 secs. As long as it does log "started" then that's all that matters I guess.

**Pull request merged**